Presentation.Web/Controllers/External/V2/ItSystemUsages/SystemRelations/SystemRelationFrequencyTypeV2Controller.cs

```diff
@@ -0,0 +1,66 @@
+﻿using Core.ApplicationServices.OptionTypes;
+using Core.DomainModel.ItSystem;
+using Core.DomainModel.ItSystemUsage;
+using Presentation;
+using Presentation.Web;
+using Presentation.Web.Controllers;
+using Presentation.Web.Controllers.External;
+using Presentation.Web.Controllers.External.V2;
+using Presentation.Web.Controllers.External.V2.ItSystemUsages;
+using Presentation.Web.Controllers.External.V2.ItSystemUsages.SystemRelations;
+using Presentation.Web.Models.External.V2;
+using Presentation.Web.Models.External.V2.Request;
+using Presentation.Web.Models.External.V2.Response;
+using Swashbuckle.Swagger.Annotations;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Web;
+using System.Web.Http;
+
+namespace Presentation.Web.Controllers.External.V2.ItSystemUsages.SystemRelations
+{
+    [RoutePrefix("api/v2/it-system-usage-relation-frequencies")]
+    public class SystemRelationFrequencyTypeV2Controller : BaseOptionTypeV2Controller<SystemRelation, RelationFrequencyType>
```

Comment:
Kald den ItSystemUsage som fornavn. Så grupperes den sammen med de andre ITSystemUsage typer i swagger og det er jo knyttet til system usage.. også selvom det er et barn på system usage men den del er en implementationsdetalje. Rodaggregaten er IT-System usage